src/utils/geometry/__tests__/removeClosePoints.test.ts

```diff
@@ -1,32 +1,47 @@
-import getExtrema from '../../../compute/getExtrema';
-import { filterPoints } from '../filterPoints';
+import { getExtrema } from '../../../compute/getExtrema';
+import { removeClosePoints } from '../removeClosePoints';
 
-test('combine minimum points after getExtrema function', () => {
+test('combine minimum points on 5x5 image', () => {
   const image = testUtils.createGreyImage([
-    [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-    [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-    [5, 5, 5, 2, 5, 5, 5, 5, 5, 5],
-    [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-    [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-    [5, 3, 3, 3, 5, 5, 5, 5, 5, 5],
-    [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-    [5, 5, 5, 5, 5, 5, 2, 5, 5, 5],
-    [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-    [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
+    [5, 5, 5, 5, 5],
+    [5, 2, 2, 2, 5],
+    [5, 5, 5, 5, 5],
+    [5, 5, 5, 5, 5],
+    [5, 0, 1, 5, 5],
   ]);
 
-  const points = getExtrema(image, { kind: 'minimum' });
+  const points = [
+    { column: 1, row: 1 },
+    { column: 1, row: 4 },
+    { column: 3, row: 1 },
+    { column: 2, row: 4 },
+  ];
 
-  const result = filterPoints(points, image, {
-    removeClosePoints: 2,
+  const result = removeClosePoints(points, image, {
+    distance: 2.5,
     kind: 'minimum',
     channel: 0,
   });
   expect(result).toStrictEqual([
-    { column: 3, row: 5 },
-    { column: 6, row: 7 },
-    { column: 3, row: 2 },
+    { column: 1, row: 4 },
+    { column: 1, row: 1 },
+  ]);
+});
+
+test('combine maximum points on 3x3 image', () => {
+  const image = testUtils.createGreyImage([
+    [1, 1, 1],
+    [1, 6, 6],
+    [1, 1, 1],
   ]);
+  const points = [
+    { column: 2, row: 1 },
+    { column: 2, row: 2 },
+  ];
+
+  const result = removeClosePoints(points, image, { distance: 2 });
```

Comment:
Would this have the same result with a value of 1?

Reply:
No, because the function checks if the euclidean distance between points is strictly smaller than the parameter value. Should it be smaller or equal?

Reply:
No I think it's better the way it is now